src/lightning/app/runners/cloud.py

```diff
@@ -240,17 +246,26 @@ def cloudspace_dispatch(
         flow_servers = self._get_flow_servers()
         network_configs = self._get_network_configs(flow_servers)
         works = self._get_works(cloudspace=cloudspace)
-        run_body = self._get_run_body(cluster_id, flow_servers, network_configs, works, False, root, True)
+        run_body = self._get_run_body(
+            cluster_id,
+            flow_servers,
+            network_configs,
+            works,
+            False,
+            root,
+            True,
+            True,
```

Comment:
lets rather use kwargs to be easier to understand what True/False is set for...

Reply:
One day :smiley: